main.js

```diff
@@ -0,0 +1,74 @@
+import {
+  renderBooks,
+  titleInput,
+  authorInput,
+  addBook,
+  localBooks,
+} from './modules/variables.js';
+
+// import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+import renderBook from './modules/render.js';
+import setStore from './modules/setStore.js';
+
+const { luxon } = window;
+const date = document.getElementById('date');
+date.innerText = luxon.DateTime.now().toFormat('MMMM dd, yyyy, hh:mm:ss a').toString();
```

Comment:
- [ ] due to the background colour and the colour of the date/time being the same this is not visible, please make sure to change the colour of the date/time so it is visible for the user without having to select the text.